libs/api/domains/cms/src/lib/models/overviewLinks.model.ts

```diff
@@ -0,0 +1,27 @@
+import { Field, ID, ObjectType } from '@nestjs/graphql'
+import { IOverviewLinks } from '../generated/contentfulTypes'
+import { Link, mapLink } from './link.model'
+import { IntroLinkImage, mapIntroLinkImage } from './introLinkImage.model'
+import { SystemMetadata } from 'api-cms-domain'
+
+@ObjectType()
+export class OverviewLinks {
+  @Field(() => ID)
+  id!: string
+
+  @Field(() => [IntroLinkImage])
+  overviewLinks?: Array<IntroLinkImage>
+
+  @Field(() => Link, { nullable: true })
+  link?: Link | null
```

Comment:
According to mapper these fields should never be undefined as they default to empty array and null.
Having them optional might not be wanted

Reply:
Right, fixed that :)